helm-chart/templates/11-ingress.yaml

```diff
@@ -6,9 +6,7 @@ metadata:
   annotations:
     certmanager.k8s.io/cluster-issuer: {{ .Values.tap.ingress.certmanager }}
     nginx.ingress.kubernetes.io/rewrite-target: /$2
-  {{- if .Values.tap.annotations }}
-    {{- toYaml .Values.tap.annotations | nindent 4 }}
-  {{- end }}
+    {{- include "kubeshark.labels" . | nindent 4 }}
```

Comment:
The labels are incorrectly templated into the `annotations` field instead of `labels` field.